sys/vm/vm_map.c

```diff
@@ -3326,7 +3326,7 @@ vm_map_protect(vm_map_t map, vm_offset_t start, vm_offset_t end,
 
 		/*
 		 * When VM_MAP_PROTECT_KEEP_CAP is used (for mprotect()),
-		 * upgrade* new_prot to include the associated VM_PROT_CAP
+		 * upgrade new_prot to include the associated VM_PROT_CAP
 		 * bits and retry the the scan.  This means that the
 		 * mprotect will fail if it spans map entries with and
 		 * without VM_PROT_CAP set.
```

Comment:
I'd clarify this comment a bit: "... with and without VM_PROT_CAP set in max_protection."